src/test/java/org/ambraproject/wombat/controller/ArticleControllerTest.java

```diff
@@ -22,114 +22,81 @@
-import java.util.List;
-import java.util.Map;
-
-import static org.junit.Assert.fail;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.*;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
-
-@ContextConfiguration(
-    classes = {SpringMvcConfiguration.class, ArticleController.class, ArticleControllerTest.class})
-@Configuration
-@DirtiesContext(classMode = ClassMode.AFTER_EACH_TEST_METHOD)
+@ContextConfiguration
 public class ArticleControllerTest extends ControllerTest {
-
-  private static final String EXPECTED_DOI = "10.1371/journal.pbio.1001091";
-
-  private static final int EXPECTED_INGESTION_NUMBER = 2;
-
-  @Bean
-  protected ArticleResolutionService articleResolutionService() {
-    final ArticleResolutionService articleResolutionService = mock(ArticleResolutionService.class);
-    return articleResolutionService;
+  @Configuration
+  static class ContextConfiguration {
+    @Bean
+    protected ArticleController articleController() {
+      return new ArticleController();
+    }
+    
+    @Bean
+    protected ArticleMetadata.Factory articleMetadataFactory(ArticleMetadata articleMetadata) {
+      return spy(new ArticleMetadata.Factory());
+    }
   }
+  
+  private static String EXPECTED_DOI = "10.1371/journal.pbio.1001091";
 
-  @Bean
-  protected ArticleService articleService() {
-    final ArticleService articleService = spy(ArticleServiceImpl.class);
-    return articleService;
-  }
+  private static int EXPECTED_INGESTION_NUMBER = 2;
 
-  @Bean
-  protected ArticleMetadata articleMetadata() throws IOException {
-    final ArticleMetadata mockArticleMetadata = mock(ArticleMetadata.class);
-    when(mockArticleMetadata.validateVisibility(anyString())).thenReturn(mockArticleMetadata);
-    when(mockArticleMetadata.populate(any(), any())).thenReturn(mockArticleMetadata);
-    when(mockArticleMetadata.fillAmendments(any())).thenReturn(mockArticleMetadata);
-    return mockArticleMetadata;
-  }
+  @Autowired
+  ArticleController articleController;
 
-  @Bean
-  protected ArticleMetadata.Factory articleMetadataFactory(ArticleMetadata mockArticleMetadata) {
-    final ArticleMetadata.Factory articleMetadataFactory = spy(new ArticleMetadata.Factory());
-    doReturn(mockArticleMetadata).when(articleMetadataFactory).newInstance(any(), any(), any(),
-        any(), any(), any());
-    return articleMetadataFactory;
-  }
+  @Autowired
+  ArticleResolutionService articleResolutionService;
 
-  @Bean
-  protected CorpusContentApi corpusContentApi() {
-    final CorpusContentApi corpusContentApi = mock(CorpusContentApi.class);
-    return corpusContentApi;
-  }
+  @Autowired
+  ArticleService articleService;
 
-  @Bean
-  protected PeerReviewService peerReviewService() {
```

Comment:
When I pulled apart ArticleController I was annoyed that this bean definition was still needed here.  Looks like you fixed that by making a spy of ArticleMetadata.Factory.  thx!